### ISSUE TITLE
Maybe the dividend payments to consumers are driving the phase transition

### ISSUE THREAD

**Issue by sbenthall** [sbenthall/SHARKFin#258]
Speaking with @mesalas today, I'm concerned that our main result is even more basic and a problem with a model design than I thought before.

Now we have a finer grained picture of the phase transition:

![image](https://github.com/sbenthall/SHARKFin/assets/68752/e4406cb5-d0aa-4ad6-8ac3-8a1ca6e02dcc)

This is the ending day of the simulation as a function of starting wealth and DPHM.
@mesalas says that the simulation crash is happening when the _institutional investors_ have sold all their shares.
This means the consumers have bought them all.
He also revealed that the institutional investors are not modeled as receiving dividend payments, whereas our consumers are.
Our consumers will be getting dividend payment that are roughly in proportion to their starting wealth.
The more starting wealth consumers have, the more they get paid as dividends.

We don't know exactly why the starting consumer wealth is leading to market crashes.
But one reason is that the consumer may be buying shares with their dividends, and this is a steady draw of shares away from the institutional investors.
Basically, _no matter what_, we would expect the market in our model to crash, eventually, because the consumers will accumulate shares and buy more with dividends.

The only counter to this I can think of is if the consumers hit their 'target wealth', and so hit some steady state.
But it's not clear that this is happening.

This issue is tightly connected to #256. Basically, we need a better grip on how the HARK agents are acting in order to design the model in a way that makes sense.

**Comment by sbenthall** [comment]
![image](https://github.com/sbenthall/SHARKFin/assets/68752/6775c3d8-3cd5-4f06-91df-f83b92bd4e37)

**Comment by alanlujan91** [comment]
![866c12ba-ad02-4149-b9f6-5fb20581fe17](https://github.com/sbenthall/SHARKFin/assets/5382704/18ff9655-401d-4918-856d-e7b14cb918fd)

Below zero, E[m'] expected cash on hand tomorrow is greater than cash today. Above zero, coh today is greater than what you expect to have tomorrow.

**Comment by sbenthall** [comment]
This is moot because we are bringing permanent income uncertainty back in.
